### PR TITLE
API-1835: adds `output-resources` to cluster-authentication operator.

### PR DESCRIPTION
**Pull request by deads2k**
now possible to properly merge.

### CODE REVIEW

**Review by bertinatto — COMMENTED** [review]
/lgtm

File: pkg/cmd/mom/output_resources_command.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	"k8s.io/cli-runtime/pkg/genericiooptions"
+)
+
+func NewOutputResourcesCommand(streams genericiooptions.IOStreams) *cobra.Command {
+	return libraryoutputresources.NewOutputResourcesCommand(RunOutputResources, streams)
+}
+
+func RunOutputResources(ctx context.Context) (*libraryoutputresources.OutputResources, error) {
+	return &libraryoutputresources.OutputResources{
+		ConfigurationResources: libraryoutputresources.ResourceList{
```
Comment (p0lyn0mial) [inline pkg/cmd/mom/output_resources_command.go:17]: I don't understand the difference between Configuration, Management and UserWorkload Resources.

File: pkg/cmd/mom/output_resources_command.go [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+			ExactResources: []libraryoutputresources.ExactResource{
+				exactResource("config.openshift.io", "ingresses", "", "cluster"),
+			},
+		},
+		ManagementResources: libraryoutputresources.ResourceList{
+			ExactResources: []libraryoutputresources.ExactResource{
+				exactResource("config.openshift.io", "clusteroperators", "", "authentication"),
```
Comment (p0lyn0mial) [inline pkg/cmd/mom/output_resources_command.go:24]: why this resource is not on the `ConfigurationResources` list?

File: pkg/cmd/mom/output_resources_command.go [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+				exactResource("", "serviceaccounts", "openshift-authentication", "oauth-openshift"),
+				exactResource("apps", "deployments", "openshift-authentication", "oauth-openshift"),
+				exactResource("oauth.openshift.io", "oauthclients", "", "openshift-browser-client"),
+				exactResource("oauth.openshift.io", "oauthclients", "", "openshift-challenging-client"),
+				exactResource("oauth.openshift.io", "oauthclients", "", "openshift-cli-client"),
+				exactResource("rbac.authorization.k8s.io", "clusterrolebindings", "", "system:openshift:openshift-authentication"),
+				exactResource("rbac.authorization.k8s.io", "rolebindings", "openshift-config-managed", "system:openshift:oauth-servercert-trust"),
+				exactResource("rbac.authorization.k8s.io", "roles", "openshift-config-managed", "system:openshift:oauth-servercert-trust"),
+			},
+			GeneratedNameResources: []libraryoutputresources.GeneratedNameResource{
```
Comment (p0lyn0mial) [inline pkg/cmd/mom/output_resources_command.go:41]: I don't understand the purpose of `GeneratedNameResources`. I checked the mom repo and it is not used.

**Pull request merged**